mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/credit/CreditApplyHandler.java

```diff
@@ -67,11 +64,7 @@ public boolean handle(MessageContext context) throws Exception {
         String clientCorrelator = null;
         String requestResourceURL = executor.getResourceUrl();
 
-        
-        FileReader fileReader = new FileReader();
-        String file = CarbonUtils.getCarbonConfigDirPath() + File.separator
-                      + FileNames.MEDIATOR_CONF_FILE.getFileName();
-		Map<String, String> mediatorConfMap = fileReader.readPropertyFile(file);
+        Map<String, String> mediatorConfMap = ConfigFileReader.getInstance().getMediatorConfigMap();
```

Comment:
Formatting

Reply:
Please check the source code.
Correctly formatted in the source. let me know if there is anything to change